.github/ISSUE_TEMPLATE/module_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Module requests
+about: For modules that you would like to see
+title: 'Module request: MODULENAME'
+labels: '9.needs: module (new)'
+assignees: ''
+
+---
+
+## Description
```

Comment:
Other templates use `###` for sections

Reply:
This is true, although semantically a second-level heading would make more sense. I'll switch this to a third-level heading and submit a separate PR to change to second-level headings.